src/css/style.css

```diff
@@ -0,0 +1,83 @@
+h1{
+	padding-top: 10px;
+	color: black;
+	text-align: center;
+	font-size: 40px;
+	font-family: verdana;
+	font-weight: bold;
+
+}
+
+button{
+	position: absolute;
+	top: 65;
+	right: 210;
+
+}
+
+#timelinePlaceholder{
+	width: 1500px;
+	border-style: solid;
+	border-width: 1px;
+	margin: auto;
+}
+
+div.tweet {
+	padding-top: 10px;
+	padding-bottom: 10px;
+	height: 110px;
+	width: 1500px;
+	border-style: solid;
+	border-bottom-width: 1px;
+	border-top-width: 0px;
+	border-right-width: 0px;
+	border-left-width: 0px;
+	border-color: lightgrey;
+}
+
+img{
+	height: 60;
```

Comment:
better to add explicit type and not just '60'.  This applies to all other similar values as well